documentation/book/assembly-upgrade.adoc

```diff
@@ -5,18 +5,81 @@
 In some cases, you can also downgrade to a lower supported Kafka version.
 
-When a newer version of {ProductName} is available, it may provide support for newer versions of Kafka. 
-Therefore, you will need to upgrade to the new version of {ProductName} before you can upgrade to a higher supported Kafka version.
-Upgrading the version of {ProductName} is done by upgrading the Cluster Operator deployment to the new version.
+{ProductName} provides support for newer versions of Kafka, but you need to upgrade {ProductName} _before_ you can upgrade to a higher supported Kafka version.
+
+*Procedure outline*
+
+In summary, upgrading {ProductName} is a two-stage process. To upgrade brokers and clients without downtime, you *must* complete the upgrade procedures in the order shown here.
+
+. First, update your Cluster Operator to the latest {ProductName} version.
+
+.. xref:assembly-upgrade-cluster-operator-{context}[]
+
+. Second, upgrade all Kafka brokers and client applications to the latest Kafka version.
+.. xref:assembly-upgrading-kafka-versions-{context}[]
 
 ifdef::StrimziUpgrades[]
-include::proc-upgrading-the-cluster-operator-0-10-0-to-0-11-0.adoc[leveloffset=+1]
+== Kafka versions
+
+The Kafka message format version and inter-broker protocol version are subject to change. As a result, the upgrade process involves making configuration changes to existing Kafka brokers and client applications (consumers and producers). The following table shows the differences between the Kafka versions:
+
+include::snip-kafka-versions.adoc[leveloffset=+1]
+
+endif::StrimziUpgrades[]
+
+ifndef::StrimziUpgrades[]
+== Kafka versions
+
+{ProductName} 1.0.0 and 1.1.0 are based on different versions of Apache Kafka:
+
+[options="header"]
+|=======================
+|{ProductName} version |Kafka version
+|1.0.0                 |2.0.0
+|1.1.0                 |2.1.1
+|=======================
+
+The Kafka message format version and inter-broker protocol version are subject to change. As a result, the upgrade process involves making both configuration changes to existing Kafka brokers and code changes to client applications (consumers and producers). The following table shows the differences between the Kafka versions:
+
+[options="header"]
+|=======================
+|Kafka version |Interbroker protocol version  |Log message format version | Zookeeper version
+|2.0.0         |2.0                           |2.0                        | 3.4.13
+|2.1.1         |2.1                           |2.1                        | 3.4.13
+|=======================
+
+Although Kafka 2.0.0 and 2.1.1 use the same version of Zookeeper, Red Hat recommends that you update your Zookeeper cluster to use the newest Zookeeper binaries before proceeding with the main {ProductName} 1.1.0 upgrade.
```

Comment:
We should not say 'Red Hat recommends' This is one area where we agreed it's okay to use the passive voice to allow docs to be shared up and downstream.  It should say "It is recommended..."  This is in the contribution guidelines.

Reply:
Updated